README.md

```diff
@@ -16,7 +16,10 @@ The structure then of the KYFB app is as follows:
 Form.io is one way to quickly create modify and update forms on the fly. Like other similar technologies, it focuses on the submission and treatment of the data for these also. However, for the scope of the KYFB and LabelBuster, this functionality is limited only to the display of the forms.
 
 ## Wrapper
-The wrapper is designed to de-couple the application away from Formio, allowing swappable form solutions in the future and to simplify the rest of the application and not have to deal with the way Formio works directly while dealing with the application logic. This helps to enforce the single responsibility principle. Much of this is configurable to cater to the individual needs of the application allowing for full re-use of this wrapper. Also
+The wrapper is designed to de-couple the application away from Formio, allowing swappable form solutions in the future and to simplify the rest of the application and not have to deal with the way Formio works directly while dealing with the application logic. This helps to enforce the single responsibility principle. Much of this is configurable to cater to the individual needs of the application allowing for full re-use of this wrapper. 
+
+Update when wrapper is moved to own repo and describe the config when its fully solidified.
```

Comment:
what would you like the new repo name to be?

Reply:
I am easy with the name, as long as we can push it to NPM and have it public.

Reply:
@DanielTurner  what name will it be?